src/pages/whats-happening/releases/index.mdx

```diff
@@ -0,0 +1,177 @@
+---
+title: Releases
+description:
+  Read about the latest updates and learn more about what the team has planned
+  for Carbon’s future.
+---
+
+import {
+  StructuredListWrapper,
+  StructuredListHead,
+  StructuredListRow,
+  StructuredListCell,
+  StructuredListBody,
+  Link,
+} from 'carbon-components-react';
+
+<PageDescription>
+
+Read about the latest updates and learn more about what the team has planned for
+Carbon’s future.
+
+</PageDescription>
+
+## Resources
+
+<Row className="resource-card-group">
+<Column colMd={4} colLg={4} noGutterSm>
+  <ResourceCard
+    subTitle="Changelog"
+    href="https://github.com/carbon-design-system/carbon/releases">
+    <MdxIcon name="github" />
+  </ResourceCard>
+</Column>
+<Column colMd={4} colLg={4} noGutterSm>
+  <ResourceCard
+    subTitle="Roadmap"
+    href="https://app.zenhub.com/workspaces/system-squad-593830641344b813db10934d/roadmap">
+
+![ZenHub icon](images/z-icon.svg)
+
+  </ResourceCard>
+</Column>
+</Row>
+
+## Release schedule
+
+Each Carbon Design System release includes library releases that are
+independently versioned following the industry-standard
+[semantic versioning](https://semver.org) specification.
+
+The Carbon team also ships minor releases every other week and when necessary,
+patch releases between minor versions to address bugs. Read more in Carbon's
+[release radar](https://github.com/carbon-design-system/carbon/wiki/Release-radar).
+
+### Carbon 10
+
+Carbon 10 was a unique release as it was the first to implement the updated IBM
+Design Language. That required product redesigns to a certain extent due to
+foundational differences between IBM Design Language versions.
+
+### Carbon 11
+
+Unlike Carbon 10, the upcoming Carbon 11 release will not require you to
```

Comment:
I feel like we're only talking about design here. I think inline theming could actually be a big effort for people to transition to from existing implementation (or its a know concern). Also we're changing our entire color token system. Neither are as big as fully branding redesign but

Reply:
@aagonzales we'll be able to talk specifics like inline theming in a new page that Jan, Josh, and I have discussed with that Carbon v11 FAQ content.